README2.md

```diff
@@ -0,0 +1,54 @@
+# Golden Sample Angular App
```

Comment:
why did you create this file? can this replace README?

Reply:
It looks like it is here only to save the data from the original file, they should be merged, right? @danielstevenberger